From 715929fcb067a3205cb9ea7d648d382b71a870bd Mon Sep 17 00:00:00 2001
From: Sean Robertson <sdrobert@cs.toronto.edu>
Date: Tue, 21 Feb 2023 15:37:27 -0500
Subject: patch

---
 setup.py | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index b112046..21140a0 100644
--- a/setup.py
+++ b/setup.py
@@ -3,14 +3,16 @@
 
 import numpy
 import setuptools
+from Cython.Build import cythonize
 
-setuptools.setup(
-    ext_modules=[setuptools.Extension(
+extensions = [setuptools.Extension(
         'libri_light_dtw',
         sources=['libriabx/libri_light/ABX_src/dtw.pyx'],
         extra_compile_args=['-O3'],
-        include_dirs=[numpy.get_include()])],
+        include_dirs=[numpy.get_include()])]
 
+setuptools.setup(
+    ext_modules=cythonize(extensions, force=True),
     # needed for cython/setuptools, see
     # http://docs.cython.org/en/latest/src/quickstart/build.html
     zip_safe=False
-- 
2.29.2.windows.2

